disnake/abc.py

```diff
@@ -1430,7 +1453,7 @@ async def send(
                     allowed_mentions=allowed_mentions,
                     message_reference=reference,
                     stickers=stickers,
-                    components=components,
+                    components=components_payload,  # type: ignore
```

Comment:
I can remove this. There's no exact reason why the linter is complaining. `components` kwarg is annotated as `List[Component]` and `components_payload`'s type is `List[ActionRow]`. ActionRow inherits from Component so I don't see any problems here. Maybe we can solve this by type hinting the return type of `components_to_dict` as `List[Component]`?

Reply:
Ah, that makes sense. We should be able to solve that by annotating the `components` kwarg as `Sequence[Component]` instead, since `List` isn't covariant iirc